app/views/task-list.html

```diff
@@ -59,7 +59,7 @@ <h2 class="app-task-list__section">
             </li>
             <li class="app-task-list__item">
               <span class="app-task-list__task-name">
-                <a class="govuk-link" href="/contact-preferences">
+                <a class="govuk-link" href="/contact-preferences" aria-describedby="company-information-completed">
```

Comment:
1. Can we update the ID so that it's correct for the section?
2. Does it matter that this'll be `aria-described-by` a non-existent element whilst this section is incomplete? Should we wrap the `aria-described-by` in a conditional?

Reply:
When I tested 2. it was fine

Reply:
Merging this for now cos we're short on time.